### DISCUSSION TITLE
Add - edit with Builderius

### DISCUSSION THREAD

**Discussion by bbk14** [builderius/builderius#107] [Ideas]
If I don't want use theme, only Builderius, please add edit with Builderius button to Pages and Posts (and all post content like custom posts or products in the future when WooCommerce will be available). After we can create or edit all post or pages from their menu like to another page builders do that

**Comment by seaj1one** [comment]
@bbk14 If this helps, I created a new page, post then I apply the template to the page. Here's a link https://docs.builderius.io/docs/templates/adding-templates

It would be interesting if you can jump to the templates like in the video, saving a few click.

**Reply by bbk14** [reply]
I understand how to do that, but I guess it will be easy or faster to start edit page after creating

**Reply by seaj1one** [reply]
I agree, it would be nice. 😀

**Comment by mrpsiho** [comment]
This is available in BETA version
